ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -88,8 +88,7 @@ public class ORSGraphHopper extends GraphHopperGtfs {
     private HashMap<Integer, Long> tmcEdges;
     private Eccentricity eccentricity;
 
-    private int minNetworkSize = 200;
-    private int minOneWayNetworkSize = 0;
+    private int minNetworkSize = 200;//TODO: does this parameter actually influence anything?
```

Comment:
yes, if you have subnetworks in the graph below this nodecount (or edgecount) they are removed from the graph.